### PR TITLE
merchants: delist kraken for GB, add note for EEA

### PR DESCRIPTION
**Pull request by nahuhh**

### CODE REVIEW

**Comment by plowsof** [comment]
if kraken still have Euros (but no direct xmr/eur) we can still have EUR, just without the *

**Comment by nahuhh** [comment]
> if kraken still have Euros (but no direct xmr/eur) we can still have EUR, just without the * 

They will have eur to ltc, but no xmr at all..

**Comment by plowsof** [comment]
ok then EUR can stay without * (the asterisk just says that they have a direct fiat/xmr pairing. e.g. there is no £GBP/xmr but they have gbp :+1:

**Review by plowsof — APPROVED** [review]

File: community/merchants/index.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ meta_descr: merchants.descr
         <h3>CEXes</h3>
         <p>{% t merchants.cexp %}</p>
         <ul class="logo">
-            <li><a href="https://www.kraken.com/">Kraken</a> (EUR*, USD*, CAD, GBP, JPY, AUD, CHF)</li>
+            <li><a href="https://www.kraken.com/">Kraken</a> (USD*, CAD, JPY, AUD, CHF)</li>
```
Comment (HardenedSteel) [inline community/merchants/index.md:96]:
```suggestion
            <li><a href="https://www.kraken.com/">Kraken</a>** (EUR*, USD*, CAD, JPY, AUD, CHF)</li>
```

Then add;
** Unavailable for EEA clients.

**Review by HardenedSteel — CHANGES_REQUESTED** [review]

**Review by HardenedSteel — APPROVED** [review]

**Comment by nahuhh** [comment]
Added note for EEA

**Pull request merged**